test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```
@@ -850,7 +867,14 @@ private <REQ, RSP> void maybeEnqueue(Message<REQ> message, InetAddressAndPort to
                             if (ctx != null)
                             {
                                 assert ctx == cb;
-                                ctx.onFailure(to, RequestFailureReason.TIMEOUT);
+                                try
+                                {
+                                    ctx.onFailure(to, RequestFailureReason.TIMEOUT);
+                                }
+                                catch (Throwable t)
+                                {
+                                    failures.add(t);
```

when IR messages retry they would NPE as they use a null `desc`... this would get handled by the future and ignored, so tests would fail saying the repair didn't complete rather than the NPE... this fix makes sure we fail fast